React/tic-tac/src/app/screens/Login/components/Form.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import PropTypes from 'prop-types';
+
+import validate from '../validate.js';
```

Comment:
You can avoid the `.js` in these imports. Also, the `validate.js` is only used in the form component, so it could be placed in the same folder as the `index.js` of such component.